### PR TITLE
Fix dispose bugs

### PR DESCRIPTION
**Pull request by johnspackman**

### CODE REVIEW

**Comment by level420** [comment]
Related to https://github.com/qooxdoo/qooxdoo/pull/168

File: framework/source/class/qx/ui/form/AbstractField.js [hunk 1]
```diff
@@ -687,8 +690,7 @@ qx.Class.define("qx.ui.form.AbstractField",
      * @return {String|null} The current value
      */
     getValue : function() {
-      var value = this.getContentElement().getValue();
-      return this.__nullValue ? null : value;
+      return this.isDisposed() || this.__nullValue ? null : this.getContentElement().getValue();
     },
```
Comment (oetiker) [inline framework/source/class/qx/ui/form/AbstractField.js]: if it is disposed the value is 'true'.
wouldn't returning null make more sense?

Comment (johnspackman) [inline framework/source/class/qx/ui/form/AbstractField.js]: oops - yes, very much an unintended result

**Comment by oetiker** [comment]
yep like it!

**Pull request merged**